Hudl.Mjolnir.Tests/TestCommands/BaseTestCommand.cs

```diff
@@ -9,8 +9,9 @@ internal abstract class BaseTestCommand<TResult> : Command<TResult>
     {
         internal BaseTestCommand() : this(TimeSpan.FromMilliseconds(10000)) { }
         internal BaseTestCommand(TimeSpan? timeout) : this("test", "test", timeout) {}
-        internal BaseTestCommand(string group, string isolationKey, TimeSpan? timeout) : this(group, null, isolationKey, timeout) {}
-        internal BaseTestCommand(string group, string name, string isolationKey, TimeSpan? timeout) : base(group, name, isolationKey, isolationKey, timeout ?? TimeSpan.FromMilliseconds(10000))
+        internal BaseTestCommand(TimeSpan? timeout,bool ignoreTimeouts=false) : this("test", "test", timeout,ignoreTimeouts) { }
```

Comment:
Your style is a bit inconsistent - sometimes there are spaces and other times not. We usually stick to [Microsoft's C# conventions](https://msdn.microsoft.com/en-us/library/ff926074.aspx) unless there's a one-off need for something different.

We've also got an [internal WIP style guide](https://github.com/hudl/docs/blob/master/styles/c-sharp.md) you can take a look at.